google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseInfo.java

```diff
@@ -129,6 +155,16 @@ public Timestamp getCreateTime() {
     return createTime;
   }
 
+  /** Returns the version retention period of the database. */
```

Comment:
nit: Could we have a more elaborate comment on this method? And also for `getEarliestVersionTime`. I don't think everyone will understand what these fields mean.

Reply:
Added more information to both params, let me know if there are satisfactory.